### PR TITLE
Add Visual C++ project, AppVeyor script

### PR DESCRIPTION
**Pull request by qmfrederik**
This completes the initial work to get libgdiplus compiling with Visual C++ and running on Windows.

Adds a Visual C++ project (and VS solution) which compiles libgdiplus. Most of the dependencies are sourced from HexChat's gtk-win32; others come from CoApp packages on NuGet.

### CODE REVIEW

**Comment by qmfrederik** [comment]
@akoeplinger This gets the msvc build up and running and adds an AppVeyor script for making sure there are no compile-time regressions.

This was "good enough" for me to develop and test #49 entirely on Windows, so I'm thinking it's worth submitting this as a PR already.

There are a couple of gotchas, though:

- On x86, I the calling convention is off - msvc libgdiplus uses cdecl instead of stdcall; but compiling with sdtcall is not straightforward because the dependencies use cdecl
- On x64, initialization fails because of a bug in the fontconfig code in gtk-win32; I worked around this on my build by just commenting out fontconfig initialization.

If you know of other projects from where I could source those dependencies, let me know.

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+environment:
+  LIBGDIPLUS_VERSION_PREFIX: 4.2
+
+before_build:
+  - cmd: choco install -y wget
+  - cmd: wget "https://dl.hexchat.net/gtk-win32/vc14/x86/gtk-Win32.7z" -O gtk-Win32.7z
+  - cmd: 7z x gtk-Win32.7z -ogtk
+  - cmd: wget "https://dl.hexchat.net/gtk-win32/vc14/x64/gtk-x64.7z" -O gtk-x64.7z
+  - cmd: 7z x gtk-x64.7z -ogtk
```
Comment (akoeplinger) [inline appveyor.yml]: why do we need gtk?

Comment (qmfrederik) [inline appveyor.yml]: We don't need GTK. GTK shares dependencies with libgdiplus (cairo, fontconfig, libpng,...), and gtk-win32 provides read-to-use win32 distribution of those dependencies.

If there's another source we can pull those dependencies from, happy to hear. Normally I try to use CoApp packages on NuGet but they are either really old or absent.

**Comment by migueldeicaza** [comment]
Thanks for the responses.   Merging

**Pull request merged**